website/static/css/custom.css

```diff
@@ -207,4 +206,17 @@ h1,h2,h3,h4,h5,h6 {
 }
 
 @media only screen and (min-width: 1500px) {
-}
\ No newline at end of file
+}
+
+.navGroupCategoryTitle {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=5373875)